src/components/SearchBar/SearchBar.styles.ts

```diff
@@ -0,0 +1,128 @@
+import { Box, TextField, IconButton } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+export const SearchContainer = styled(Box)(({ theme }) => ({
+  position: 'absolute',
+  width: '324px',
+  height: '67px',
+  left: '929px',
+  top: '91px',
+  background: 'rgba(250, 250, 250, 0.75)', // Fondo con opacidad
+  backdropFilter: 'blur(10px)', // Añadir blur para efecto vidrio
+  borderRadius: '15px',
+  display: 'flex',
+  alignItems: 'center',
+  boxSizing: 'border-box',
+  border: '1px solid rgba(255, 255, 255, 0.2)', // Borde sutil
+
+  [theme.breakpoints.down('sm')]: {
+    width: '342px',
+    height: '67px',
+    left: '24px',
+    top: '231.87px',
+  },
+}));
+
+export const SearchIconButton = styled(IconButton)(({ theme }) => ({
+  position: 'absolute',
```

Comment:
SearchIconButton uses absolute positioning and very small hard-coded px sizes and offsets. This makes the component fragile and non-themeable. Use theme.spacing and relative placement within a flex container instead.

**Sugerencia:**
```typescript
Place the icon in-flow (no absolute) and use padding/margin derived from theme.spacing. Example: display:flex; align-items:center; paddingLeft: theme.spacing(1); then set the IconButton sizes using theme.spacing values.
```